cts/lab/CTSscenarios.py

```diff
@@ -4,7 +4,11 @@
 __copyright__ = "Copyright 2000-2020 the Pacemaker project contributors"
```

Comment:
Is there a strategy when we update the copyright-date? Or every time we touch a file?

Reply:
Whoops, every time -- fixed